medusa/classes.py

```diff
@@ -129,15 +130,17 @@ def __init__(self, item, download_current_quality):
         self.actual_episodes = None
 
     def add_result_to_cache(self, cache):
-        # Cache the item if needed
+        """Cache the item if needed."""
```

Comment:
Same with this docstring.  What type of item. When would it be needed?  A better example would be "Cache a search result if not already cached."